CMake/SiloConfig.cmake.in

```diff
@@ -0,0 +1,79 @@
+#-----------------------------------------------------------------------------
+# SILO Config file for compiling against SILO build directory
+#-----------------------------------------------------------------------------
+@PACKAGE_INIT@
+
+set(SILO_PACKAGE_NAME "Silo")
+
+set (${SILO_PACKAGE_NAME}_VALID_COMPONENTS static shared)
+
+#-----------------------------------------------------------------------------
+# User Options
+#-----------------------------------------------------------------------------
+set (SILO_BUILD_SHARED_LIBS    @BUILD_SHARED_LIBS@)
+
+set(SILO_ENABLE_SILEX @SILO_ENABLE_SILEX@)
+set(SILO_ENABLE_BROWSER @SILO_ENABLE_BROWSER@)
+set(SILO_ENABLE_FORTRAN @SILO_ENABLE_FORTRAN@)
+set(SILO_ENABLE_HDF5 @SILO_ENABLE_HDF5@)
+set(SILO_ENABLE_JSON @SILO_ENABLE_JSON@)
+set(SILO_ENABLE_PYTHON_MODULE @SILO_ENABLE_PYTHON_MODULE@)
+set(SILO_ENABLE_NETCDF @SILO_ENABLE_NETCDF@)
```

Comment:
We don't need an enabler for netCDF.

Reply:
Okay, will fix that here and in root CMakeLists.txt